### DISCUSSION TITLE
Suggestion: deep link PocketBase docs

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#2696] [Q&A]
I just noticed that I can link to an API section such as https://pocketbase.io/docs/api-records/, but there is no way to a specific API call such as `
/api/collections/collectionIdOrName/confirm-password-reset`.

When I navigate to a specific accordion, the browser URL doesn't change

https://github.com/pocketbase/pocketbase/assets/1068356/de217002-e397-4e34-b997-1e3eeb8d7c50

**Comment by ganigeorgiev** [comment]
I've added it to my notes.
This will be improved soon with the upcoming docs refactoring (_I'm currently experimenting with the goja app hooks integration and trying to automate the Go -> ambient TS types generation_).
